packages/babel-plugin-transform-spread/test/fixtures/spread/known-rest/expected.js

```diff
@@ -3,5 +3,5 @@ function foo() {
     bar[_key] = arguments[_key];
   }
 
-  return [].concat(bar);
```

Comment:
for this case it's not even necessary to call concat at all right?

Reply:
We have to create a new array from it so yeah. Because the original call was `return [...bar];`

Reply:
cool I wrote too soon haha